sample-kotlin/versions.properties

```diff
@@ -1,6 +1,6 @@
 #### Dependencies and Plugin versions with their available updates.
-#### Generated by `./gradlew refreshVersions` version 0.40.2-SNAPSHOT
-#### Revision of dependency notations removals: 9
+#### Generated by `./gradlew refreshVersions` version 0.41.0-SNAPSHOT
```

Comment:
Should the new generated version be `0.40.3-SNAPSHOT`?

Reply:
No, `plugins/versions.txt` contains `0.41.0-SNAPSHOT`